fastly/stats_historical.go

```diff
@@ -147,14 +146,19 @@ func (c *Client) GetStatsField(i *GetStatsInput) (*StatsFieldResponse, error) {
 
 // GetStatsJSON fetches stats and decodes the response directly to the JSON struct dst.
 func (c *Client) GetStatsJSON(i *GetStatsInput, dst any) error {
-	p := "/stats"
+	components := make([]string, 1, 5)
```

Comment:
Why 5? Looks like 3 total here.

Reply:
It can be 5 if both `i.Service` and `i.Field` are non-nil.